interface/wx/notebook.h

```diff
@@ -191,6 +191,19 @@ class wxNotebook : public wxBookCtrlBase
     */
     virtual void SetPadding(const wxSize& padding);
 
+    /**
+        Returns wxTOP / wxBOTTOM / wxRIGHT / wxLEFT
+    */
+    wxDirection GetTabOrientation() const;
+
+    /**
+        Return the rectangle in window coordinates of the page selector.
+        For example, for a wxNotebook this would be its tab.
```

Comment:
I'm not sure it makes sense to say "for a wxNotebook" in wxNotebook documentation. Was this supposed to be a `wxBookCtrlBase` function initially?